### DISCUSSION TITLE
433Mhz radio remote control mode

### DISCUSSION THREAD

**Discussion by Bill-Dung** [OMOTE-Community/OMOTE-Firmware#56] [Ideas]
What I have always missed with universal remote controls is a 433 MHz transmitter to switch radio sockets or radio Christmas tree lights. It would be great if OMOTE could be expanded in this regard. Then I would finally only have a single remote control on the table.

**Comment by Dark1886** [comment]
Although maybe not the most perfect solution, but you could have a 433MHz bridge in your home and just trigger a command using an MQTT message. This does require another device in the home, but still leave only a single remote on the table.
